lms/templates/student_account/register.underscore

```diff
@@ -39,31 +39,35 @@
                     </button>
                 <% } %>
             </div>
-            <div class="section-title lines">
-                <h3>
-                    <span class="text"><%- gettext("or create a new one here") %></span>
-                </h3>
-            </div>
-        <% } else { %>
+            <% if (!context.is_require_third_party_auth_enabled) { %>
```

Comment:
It looks like this will remove the registration form if third party auth is required. Is that accurate? If so then we need to change this template to simply hide the form, either by setting the display attribute to hidden or hiding it with CSS. The registration form is still used for creating accounts even when using third party auth. It is filled and submitted using javascript as part of the account creation flow after the user authenticates with the third party. If the form is not present on the page then the user is never able to create an account.